src/test/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthServiceTest.java

```diff
@@ -343,12 +350,48 @@ void GIVEN_GG_with_cda_WHEN_updated_ca_type_THEN_ca_is_updated()
         assertThat(initialCA, not(thirdCA));
         assertThat(getCaPassphrase(), not(initialCaPassPhrase));
 
-        verify(client, times(3)).putCertificateAuthorities(putCARequestArgumentCaptor.capture());
+        verify(client, times(2)).putCertificateAuthorities(putCARequestArgumentCaptor.capture());
         List<List<String>> certificatesInRequests =
                 putCARequestArgumentCaptor.getAllValues().stream().map(
                         PutCertificateAuthoritiesRequest::coreDeviceCertificates).collect(
                         Collectors.toList());
-        assertThat(certificatesInRequests, contains(initialCACerts, secondCACerts, thirdCACerts));
+        assertThat(certificatesInRequests, contains(initialCACerts, thirdCACerts));
+    }
+
+    @Test
+    void GIVEN_certificateAuthorityConfiguration_WHEN_itChanges_THEN_CAisConfigured() throws InterruptedException,
```

Comment:
This is the new test that checks that we are calling the use case when the certificateAuthority configuration changes.